roadmap/implementers-guide/src/node/disputes/participation.md

```diff
@@ -104,7 +103,7 @@ Nothing to store, everything lives in `VotesDB`.
 		1. else:
 			1. iff: result is `Detection`
 				1. Request `AvailableData` via `RecoverAvailableData` to obtain the `PoV`.
-				1. Call `ValidateFromChainState` to validate the `PoV` with the runtime logic
+				1. Call `ValidateFromChainState` to validate the `PoV` with the runtime validation logic
```

Comment:
I believe we would typically want to call `ValidateFromExhaustive` because we have requested the `AvailableData`.